internal/configs/version2/nginx-plus.virtualserver.tmpl

```diff
@@ -68,8 +68,18 @@ match {{ $m.Name }} {
 {{ end }}
 
 server {
+
+    {{ if $.InternalRouteServer }}
+    listen 443 ssl;
+    {{if not $s.DisableIPV6}}listen [::]:443 ssl;{{end}}
+    ssl_certificate /etc/nginx/secrets/spiffe_cert.pem;
+    ssl_certificate_key /etc/nginx/secrets/spiffe_key.pem;
+    {{end}}
+
+    {{ if not $.InternalRouteServer}}
```

Comment:
could just do an "else" here instead of a new if statement.

Reply:
Is there a preferred pattern for templating logic? If not the `if not` block seems more clear.

Reply:
I would just write it based on the usual if/else standards.

if internalRouteServer is set:
...
otherwise (else) it's not set:
...

Reply:
Updated.